--- insert_and_print/example.ecf	2016-11-25 10:01:29.000000000 -0500
+++ Eiffel_17.05/unstable/library/persistency/database/sqlite3/examples/insert_and_print/example.ecf	2017-02-05 16:37:06.625782245 -0500
@@ -14,7 +14,7 @@
 			<concurrency use="none"/>
 		</capability>
 		<library name="base" location="$ISE_LIBRARY\library\base\base.ecf" readonly="true"/>
-		<library name="sqlite" location="..\..\sqlite.ecf" readonly="false"/>
+		<library name="sqlite" location="$ISE_LIBRARY/unstable/library/persistency/database/sqlite3/sqlite.ecf" readonly="false"/>
 		<cluster name="example" location=".\">
 			<file_rule>
 				<exclude>/.svn$</exclude>
