--- liteterm.ecf	2016-11-23 17:59:01.000000000 -0500
+++ Eiffel_18.01/unstable/library/persistency/database/sqlite3/examples/liteterm/liteterm.ecf	2017-02-05 16:37:49.322124728 -0500
@@ -17,7 +17,7 @@
 		<library name="arg_parser" location="$ISE_LIBRARY\library\argument_parser\argument_parser.ecf"/>
 		<library name="base" location="$ISE_LIBRARY\library\base\base.ecf" readonly="false"/>
 		<library name="base_extension" location="$ISE_LIBRARY\library\base_extension\base_extension.ecf"/>
-		<library name="sqlite" location="..\..\sqlite.ecf"/>
+		<library name="sqlite" location="$ISE_LIBRARY/unstable/library/persistency/database/sqlite3/sqlite.ecf"/>
 		<library name="testing" location="$ISE_LIBRARY\library\testing\testing.ecf"/>
 		<cluster name="liteterm" location=".\" recursive="true"/>
 	</target>
