--- sqlite-safe.ecf	2012-01-11 10:01:42.000000000 -0500
+++ Eiffel_15.12/unstable/library/persistency/database/sqlite3/sqlite-safe.ecf	2013-09-26 21:28:53.005649578 -0400
@@ -36,6 +36,11 @@
 				<multithreaded value="true"/>
 			</condition>
 		</external_library>
+		<external_linker_flag value="-lsqlite3">
+			<condition>
+				<platform excluded_value="windows"/>
+			</condition>
+		</external_linker_flag>
 		<external_library location="$(ISE_LIBRARY)\unstable\library\persistency\database\sqlite3\spec\$(ISE_C_COMPILER)\$(ISE_PLATFORM)\lib\esqlite.lib">
 			<condition>
 				<platform value="windows"/>
